src/datumaro/experimental/fields.py

```diff
@@ -262,22 +262,32 @@ class LabelField(Field):
     semantic: Semantic
     dtype: Any
     multi_label: bool = False  # Flag to indicate if this field should handle multi-labels
+    is_list: bool = False
+
+    @property
+    def _pl_type(self) -> pl.DataType:
+        pl_type = self.dtype
+        if self.multi_label:
+            pl_type = pl.List(pl_type)
+        if self.is_list:
+            pl_type = pl.List(pl_type)
```

Comment:
```suggestion
        if self.multi_label or self.is_list:
            pl_type = pl.List(pl_type)
```

Reply:
This is not the same because in the case where both multi_label and is_list are true, `pl.List` should be applied twice leading to the type: `pl.List(pl.List(self.dtype))`. In that case, we will be missing a pl.List if we merge the if-clauses.